packages/core-utils/src/time.js

```diff
@@ -88,34 +82,27 @@ export function formatSecondsAfterMidnight(seconds, timeFormat) {
  */
 export function getUserTimezone() {
   if (process.env.NODE_ENV === "test") return process.env.TZ;
-  // FIXME There is an issue with tz.guess being undefined that has not yet been
-  // resolved. https://github.com/opentripplanner/otp-ui/issues/152
-  if (!moment.tz || typeof moment.tz.guess !== "function") {
-    // eslint-disable-next-line no-console
-    console.warn(
-      "Error guessing user's timezone (moment.tz or moment.tz.guess not defined). Defaulting to America/New_York."
-    );
-    return "America/New_York";
-  }
-  return moment.tz.guess();
+  return Intl.DateTimeFormat().resolvedOptions().timeZone;
 }
 
 /**
  * Formats current time for use in OTP query
  * The conversion to the user's timezone is needed for testing purposes.
  */
 export function getCurrentTime(timezone = getUserTimezone()) {
-  return moment()
-    .tz(timezone)
-    .format(OTP_API_TIME_FORMAT);
+  return format(
+    utcToZonedTime(new Date(Date.now()), timezone),
+    OTP_API_TIME_FORMAT
+  );
 }
 
 /**
  * Formats current date for use in OTP query
  * The conversion to the user's timezone is needed for testing purposes.
  */
 export function getCurrentDate(timezone = getUserTimezone()) {
-  return moment()
-    .tz(timezone)
-    .format(OTP_API_DATE_FORMAT);
+  return format(
+    utcToZonedTime(new Date(Date.now()), timezone),
```

Comment:
Isn't doing `new Date(Date.now())` redundant? Wouldn't `new Date()` suffice?

Reply:
Using `new Date()` broke the unit tests, but shortening to `Date.now()` works...